priv/templates/components/image.eex

```diff
@@ -88,6 +97,14 @@ defmodule <%= @module %> do
         "max-w-full",
         rounded_size(@rounded),
         shadow_size(@shadow),
+        blur(@blur),
+        brightness(@brightness),
+        contrast(@contrast),
+        hue_rotate(@hue_rotate),
```

Comment:
This section, like attributes, should also be consolidated into a function with two inputs.